erpnext/accounts/report/accounts_payable/accounts_payable.js

```diff
@@ -28,6 +28,12 @@ frappe.query_reports["Accounts Payable"] = {
 				});
 			}
 		},
+		{
+			"fieldname":"supplier_type",
+			"label": __("Supplier Type"),
+			"fieldtype": "Link",
+			"options": "Supplier Type"
```

Comment:
"Supplier Type" has been renamed to "Supplier Group" in develop branch.

Reply:
thats ok i have applied this requests